code/modules/capture_the_flag/ctf_panel.dm

```diff
@@ -0,0 +1,51 @@
+GLOBAL_DATUM_INIT(ctf_panel, /datum/ctf_panel, new())
+
+/datum/ctf_panel
+
+/datum/ctf_panel/ui_state(mob/user)
+	return GLOB.observer_state
+
+/datum/ctf_panel/ui_interact(mob/user, datum/tgui/ui)
+	ui = SStgui.try_update_ui(user, src, ui)
+	if(!ui)
+		ui = new(user, src, "CTFPanel")
+		ui.open()
+
+/datum/ctf_panel/ui_data(mob/user)
+	var/list/data = list()
+	data["teams"] = list()
+	data["enabled"] = ""
+	for(var/obj/machinery/capture_the_flag/team in GLOB.machines)
+		var/list/this = list()
+		this["name"] = team
+		this["color"] = team.team
+		this["score"] = team.points + team.control_points
+		this["team_size"] = team.team_members.len
+		this["refs"] += "[REF(team)]"	
+		data["teams"] += list(this)
+		if(team.ctf_enabled)
```

Comment:
You're setting this once for every team, so if one team marks CTF as not enabled it'll say none of them are. I assume this causes no problems in practice, since at the moment everything uses one CTF controller, but it's a bit funky to read, you should only be setting this once, even just wrapping this in `if(!data["enabled"])` would be good.